.github/workflows/all_plugins.yaml

```diff
@@ -16,7 +16,7 @@ on:
 
 jobs:
   analyze:
-    timeout-minutes: 30
+    timeout-minutes: 90
```

Comment:
Let's try out if a higher timeout solves the `melos boostrap` issue. I also created a ticket in the Melos repo (I don't know if this is a Flutter or Melos issue): https://github.com/invertase/melos/issues/322

Reply:
I feel like it could just hide the underlying issue. Maybe distributing the analysis between multiple GitHub action job would be more efficient? 
Also for a 15% flaky test, a 3 time increase seems a lot. Was it still flaky with smaller increase?

Reply:
I will try to find a better solution 👍 @blaugold wrote a comment in https://github.com/invertase/melos/issues/322#issuecomment-1156426868

Reply:
Setting `runPubGetInParallel` to `false` in Melos version `2.4.0` seems to fix the `melos bootstrap` issue 👍  Therefore I reverted the timeout increases. But for the `analyze` job I would still suggest increasing the timeout a bit (to 45 minutes) because I had some runs where the run took 25 minutes.